src/@next/Switch/SwitchStyle.tsx

```diff
@@ -42,7 +67,7 @@ export const SwitchStyle = styled.div`
   }
 `;
 
-export const InputStyle = styled.input`
+export const InputStyle = styled.input<SwitchProps>`
```

Comment:
I dont think SwitchProps will be needed anymore

Reply:
removed